src/app/product-details-page/components/product-description/product-image-preview/zoomit/zoomit.component.ts

```diff
@@ -0,0 +1,38 @@
+import { Component, OnInit, Input, OnChanges } from '@angular/core';
+
+@Component({
+  selector: 'app-zoomit',
+  templateUrl: './zoomit.component.html',
+  styleUrls: ['./zoomit.component.scss']
+})
+export class ZoomitComponent implements OnInit, OnChanges {
+  // @Input () zsrc: string;
+  zsrc = 'http://www.therxreview.com/wp-content/uploads/2013/05/Reebok-Track-Jacket.png';
+  public pos: string;
+  public zUrl: string;
+  isHovered = false;
+
+  ngOnInit() {
+  this.zUrl = `url(${this.zsrc})`;
```

Comment:
why do you need additional zUrl property?

zsc is the same

Reply:
wrapping in the url should be in the template

try `[style.background-image.url]="zUrl"`